Index: Backend/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Environments\r\n.env\r\nenv/\r\n.venv\r\nvenv/\r\n\r\n# Django stuff:\r\n*.log\r\nlocal_settings.py\r\ndb.sqlite3\r\ndb.sqlite3-journal\r\n\r\n# Distribution / packaging\r\n.Python\r\n\r\n# mkdocs documentation\r\n/site
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Backend/.gitignore b/Backend/.gitignore
--- a/Backend/.gitignore	(revision 321dca0cd96f355230c53c5be9622b1d88f5c636)
+++ b/Backend/.gitignore	(date 1695243726936)
@@ -1,3 +1,7 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+
 # Environments
 .env
 env/
@@ -5,7 +9,6 @@
 venv/
 
 # Django stuff:
-*.log
 local_settings.py
 db.sqlite3
 db.sqlite3-journal
@@ -14,4 +17,14 @@
 .Python
 
 # mkdocs documentation
-/site
\ No newline at end of file
+/site
+
+# Logs
+*.log
+log/
+
+# Django migration files
+*/migrations/
+
+# PyCharm
+.idea/
\ No newline at end of file
